doc/requirements.txt

```diff
@@ -22,6 +22,7 @@ sphinx-automodapi
 exhale>=0.3.3
 graphviz
 pybind11
+nanobind
```

Comment:
Is this a doc specific requirement? I'm just trying to think why it might show up here since we don't include any of our other build requirements (which is good, we wan't to avoid duplication whenever possible)

Reply:
Good catch! I think I had added this in when trying to get the docs build to work, but the real issue was something else. I removed nanobind from this list (in 1333f83) and the docs still build, so indeed it wasn't necessary. Thanks!